folium/plugins/timestamped_geo_json.py

```diff
@@ -9,6 +9,29 @@
 
 from jinja2 import Template
 
+_default_js = [
+    ('jquery2.0.0',
+     'https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.0/jquery.min.js'),
+    ('jqueryui1.10.2',
+     'https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.10.2/jquery-ui.min.js'),
+    ('iso8601',
+     'https://rawcdn.githack.com/nezasa/iso8601-js-period/master/iso8601.min.js'),
+    ('leaflet.timedimension',
+     'https://rawcdn.githack.com/socib/Leaflet.TimeDimension/master/dist/leaflet.timedimension.min.js'),  # noqa
+    ('moment',
+     'https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js')
+]
+
+_default_css = [
+    ('highlight.js_css',
+     'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/default.min.css'),
+    ('leaflet.timedimension_css',
+     'https://rawcdn.githack.com/socib/Leaflet.TimeDimension/master/dist/leaflet.timedimension.control.min.css'),
+    # noqa
+    ('bootstrap_theme_css',
+     'https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css')
```

Comment:
I don't see this one in the plugin. It's in `folium.py` though. Maybe a copy-paste error?

Reply:
Yeah looks like simple oversight on my part. Deleted Bootstrap CSS theme.